frontend/packages/knative-plugin/src/components/add/EventSource.tsx

```diff
@@ -33,9 +39,10 @@ const EventSource: React.FC<Props> = ({
   activeApplication,
   contextSource,
 }) => {
-  const typeEventSource = EventSources.CronJobSource;
+  const typeEventSource = KnownEventSources.CronJobSource;
```

Comment:
Why do you need to rename this `enum`? This creates unnecessary changes all across the PR.

Reply:
I thought `KnownEventSources` should be more descriptive for a name

Reply:
i think we should have what;s there currently i.e `EventSources` we can do just this refactor later if need be as in this PR it brings lot of other chnages. WDYT?

Reply:
Yes. we can change the name if required in a separate PR
reverted to `EventSources`